libs/hdf-converters/src/trufflehog-mapper.ts

```diff
@@ -0,0 +1,110 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {
+  BaseConverter,
+  ILookupPath,
+  MappedTransform
+} from './base-converter';
+
+export class TrufflehogMapper extends BaseConverter {
+  withRaw: boolean;
+
+  mappings: MappedTransform<
+    ExecJSON.Execution & {passthrough: unknown},
+    ILookupPath
+  > = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: null  
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null  
+    },
+    profiles: [
+      {
+        name: {path: 'SourceName'},              
+        title: {path: 'SourceName'},           
```

Comment:
```suggestion
        name: {path: 'wrapper[0].SourceName'},              
        title: {path: 'wrapper[0].SourceName'},           
```
Need to access the first element of the wrapper array field.